### PR TITLE
Add django 4.0 and python 3.10 compatibility and tests

### LINKED ISSUE #190: Breaks on django 4.0
Currently, this library breaks when querying on django 4.0 due to the use of the new elide_empty keyword

image

### PR DESCRIPTION
**Pull request by moshfrid**
closes #190

### CODE REVIEW

File: .dccache [hunk 1]
```diff
@@ -0,0 +1 @@
...54,"48074a24d554fecda61acfef85ad8faad6cc46cc959568a98c2769e00ee3812f"],"c:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/tests/test_soft_delete_cascade.py":[6337,1639462933996.0156,"e9cd0f1fa8cf02dce625ca4ce13e5ef24bed61d74c9ef2e6d720eaf1697071fc"],"c:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/tests/test_subquery.py":[1393,1639462933997.0176,"32e55d828929f7d48bb1de0675cd79f7e06c358dd5a25818b1abe94f61e4d78f"],"c:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/tests/test_visible_by_pk.py":[2824,1639462933997.0176,"a0613d389d31ece51433211577506d6505c526f21afbfdbccd22ba2ce2aba5d4"],"c:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/tests/urls.py":[568,1639462933998.1387,"9deae4c1e3634ab7922bd5121bee20a4e3b6643ff04c368915164038aaefefe6"],"c:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/tests/__init__.py":[0,1639462933985.0159,"e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"],"c:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/templates/safedelete/undelete_selected_confirmation.html":[846,1639462933984.0146,"9d38b0e2d4e1d1ecd776b2a94b5579e0a23e253d7801bdb97811ba3fea28e78f"]}
```
Comment (Gagaro) [inline .dccache]: Don't commit that file (you can add it to .gitignore)

**Review by Gagaro — CHANGES_REQUESTED** [review]
Thanks for the PR :+1: , just a file to remove and I'll also let the tests run.

File: .dccache (hunk 1 above)
Comment (moshfrid) [reply .dccache]: done. Thanks!

**Review by Gagaro — APPROVED** [review]

**Comment by moshfrid** [comment]
Thanks! When do you expect the next release?

**Comment by Gagaro** [comment]
Released just now in 1.1.2.

**Pull request merged**